FluentUI/src/main/res/layout/view_list_item.xml

```diff
@@ -33,14 +34,37 @@
             android:textAppearance="@style/TextAppearance.FluentUI.ListItemTitle"
             tools:text="Title" />
 
-        <TextView
-            android:id="@+id/list_item_subtitle"
+        <android.support.constraint.ConstraintLayout
+            android:id="@+id/list_item_subtitle_view_container"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/fluentui_list_item_text_vertical_spacing"
-            android:textAppearance="@style/TextAppearance.FluentUI.ListItemSubtitle"
-            tools:text="Subtitle"
-            tools:visibility="visible" />
+            android:layout_marginTop="@dimen/fluentui_list_item_text_vertical_spacing">
+
+            <TextView
+                android:id="@+id/list_item_subtitle"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textAppearance="@style/TextAppearance.FluentUI.ListItemSubtitle"
+                app:layout_constraintTop_toTopOf="@+id/list_item_subtitle_view_container"
```

Comment:
Should we not use, maxLines and ellipsize somewhere? 
Have you tested it for a very long text?

Reply:
This is using previous subtitle only which provides configurable maxLines and ellipsize properties. I have just added one more custom view beside the subtitle.

Reply:
"_ellipsize is happening in the start_" This is just an example. Since it is the same subtitle which was present earlier as well...ellipsize and maxLines are fully configurable.